int16 id
int64 xmin
int64 ymin
int64 xmax
int64 ymax
float32 x
float32 y
float32 vx
float32 vy
float64 probability
string Class